server/models/Transaction.js

```diff
@@ -498,6 +505,8 @@ export default (Sequelize, DataTypes) => {
     await Transaction.createDoubleEntry(donationTransaction);
 
     // Remove fees from main transaction
+    transaction.paymentProcessorFeeInHostCurrency =
+      transaction.paymentProcessorFeeInHostCurrency - paymentProcessorFeeInHostCurrency;
```

Comment:
`paymentProcessorFeeInHostCurrency - paymentProcessorFeeInHostCurrency;` is difficult to read.

Can you consider renaming variables?